serverless/event_workflows/serverless-triggers.adoc

```diff
@@ -0,0 +1,28 @@
+[id="serverless-triggers"]
+= Filtering events using triggers
+include::modules/common-attributes.adoc[]
+include::modules/serverless-document-attributes.adoc[]
+:context: serverless-triggers
+
+toc::[]
+
+Using triggers enables you to filter events from the broker for delivery to event sinks.
+
+[id="serverless-triggers-prereqs"]
+== Prerequisites
```

Comment:
Should the other topics in the Knative Engineering have global prerequisites also?

Reply:
Probably but I'll update these as I make changes to the other sections since it would probably be easier to create a module for this and include it, and I don't want to widen the scope of this PR too much for now. Is that acceptable?

Reply:
Added an issue to track so I don't forget: https://issues.redhat.com/browse/SRVCOM-1288
Will come back to this one when I have more bandwidth to check all the procedures